core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -168,6 +236,50 @@ protected void doStop() throws Exception {
     return server;
   }
 
+  private static String[] getList(String original) {
+    String[] parts = original.split(",");
+    for (int i = 0; i < parts.length; i++) {
+      parts[i] = parts[i].trim();
+    }
+    return parts;
+  }
+
+  // TODO: delete deprecatedPort parameter when `PORT_CONFIG` is deprecated. It's only used to support the deprecated
```

Comment:
Interesting, is this deprecation going to be logged anywhere yet? We'll definitely need to update any documentation for any projects that use rest-utils to indicate this transition would need to be made. If we're not logging anything yet, that might be fine too; maybe just suggest that using `listeners` is preferred, but `port` is still supported for compatibility.